drivers/tapdisk-utils.h

```diff
@@ -100,6 +100,6 @@ shm_create(struct shm *shm);
 int
 shm_destroy(struct shm *shm);
 
-inline long long timeval_to_us(struct timeval *tv);
+long long timeval_to_us(struct timeval *tv);
```

Comment:
Ewwww.  The way to do this nicely is to delete everything from tapdisk-util.c and have

```c
static inline uint64_t timeval_to_us(const struct timeval *tv)
{
    return (uint64_t)tv->tv_sec * 1000000 + tv->tv_usec;
}
```

Calling into a separate TU for a multiplication like this is mad.